lib/active_admin/helpers/settings.rb

```diff
@@ -22,6 +22,10 @@ def self.included(base)
     end
 
     def read_default_setting(name)
+      if (value = default_settings[name]).respond_to?(:call)
```

Comment:
Meh.  The more I look at the implementation of application settings the less I like it.  I would like a little time to submit a counter proposal.